### ISSUE TITLE
Backport 1.14 Resolution/Refresh Rate Features

### ISSUE THREAD

**Issue by da3dsoul** [sp614x/optifine#3255]
I don't know what your rules on backports are, if you even consider them for anything but fatal bugs.
I have an LG B9 OLED. It's a big, 4k 120Hz HDMI 2.1 TV. Needless to say, HDMI 2.0b is what my GTX1060 has, so it's 4K60 or 1080p120. Some games handle that better than others, so I'm kind of used to it clamping me to 60Hz even for 1080p.
In 1.12, my only option is to attempt to play 1080p with whatever refresh rate it decides on (60 with Optifine...4K without). That was okay, and I was going to live with it. Then I tried a 1.14 pack, and it's hard to go back, as it let me appropriately tell this fabulous TV to run at the 1080p120 that it should. I would just let it go to 4k, but I can't read any of the UI, and I can run it at 40fps...sometimes.

TL;DR
Can you please backport the Fullscreen Resolution/Refresh Rate Selector from 1.14 to 1.12?
I can usually play the game without it, but it actually would make some mod packs playable when they weren't before. Other TVs with less intelligent EDIDs would benefit even more.

**Comment by McHox** [comment]
just set the preferred refresh rate in the nvcp to max available and it should override whatever is set ingame

**Comment by da3dsoul** [comment]
Thanks for the advice. Sadly, that setting doesn't work for this (also not a thing on my other machine with an RX570 afaik). I have outrageous tearing on 1080p, which leads me to believe that it's actually running at lower than 60Hz, as well. My TV doesn't have an OSD section that reports the current refresh rate, so it's hard to say for sure. I sent them an email about that one, considering it's a GSYNC certified TV...and $2000.
On the 1.14 version, I don't have tearing, and it's just visibly way smoother (and actually says the refresh rate in the video settings).
